components/match2/commons/starcraft_starcraft2/match_summary_starcraft_ffa.lua

```diff
@@ -9,470 +9,132 @@
-					end
-					local icon, color = StarcraftMatchSummaryFfa._getTrophy(opponent.placement)
-					return mw.html.create()
-							:tag('i')
-								:addClass('panel-table__cell-icon')
-								:addClass(icon)
-								:addClass(color)
-								:done()
-							:tag('span'):addClass('panel-table__cell-game__text')
-									:wikitext(placementDisplay):done()
-				end,
-			},
-		},
-		{
-			class = 'panel-table__cell__game-kills',
-			iconClass = 'fas fa-star',
-			show = {
-				value = function(match)
-					return not Logic.readBool(match.noScore)
-				end
-			},
-			header = {
-				value = 'Points',
-				mobileValue = 'Pts.',
-			},
-			row = {
-				value = function (opponent)
-					return OpponentDisplay.InlineScore(Table.merge({extradata = {}, score = ''}, opponent))
-				end,
-			},
-		},
-	}
-}
-
----@param props {match: StarcraftMatchGroupUtilMatch, bracketResetMatch: StarcraftMatchGroupUtilMatch?, config: table?}
----@return Html
-function StarcraftMatchSummaryFfa.getByMatchId(props)
-	local match = StarcraftMatchSummaryFfa._opponents(props.match)
-
-	return StarcraftMatchSummaryFfa._createOverallPage(match)
-end
-
-function StarcraftMatchSummaryFfa._opponents(match)
-	-- Add match opponent data to game opponent and the other way around
-	Array.forEach(match.games, function (game)
-		game.extradata.opponents = Array.map(game.opponents, function (opponent, opponentIdx)
-			return Table.merge(opponent, {
-				placement = opponent.placement,
-				status = opponent.status or 'S',
-				score = opponent.score or (game.scores or {})[opponentIdx],
-			})
-		end)
-	end)
-
-	Array.forEach(match.opponents, function (opponent, opponentIdx)
-		opponent.games = Array.map(match.games, function (game)
-			return game.extradata.opponents[opponentIdx]
-		end)
-	end)
-
-	local placementSortFunction = function(opponent1, opponent2)
-		local place1 = opponent1.placement or math.huge
-		local place2 = opponent2.placement or math.huge
-		if place1 ~= place2 then
-			return place1 < place2
-		end
-		if opponent1.status ~= 'S' and opponent2.status == 'S' then
-			return false
-		end
-		if opponent2.status ~= 'S' and opponent1.status == 'S' then
-			return true
-		end
-		if opponent1.score and opponent2.score and opponent1.score ~= opponent2.score then
-			return opponent1.score > opponent2.score
-		end
-		return (opponent1.name or ''):lower() < (opponent2.name or ''):lower()
-	end
-
-	-- Sort match level based on placement
-	Array.sortInPlaceBy(match.opponents, FnUtil.identity, placementSortFunction)
-
-	if not match.finished then
-		return match
-	end
-	local cache = {
-		index = #match.opponents,
-		currentWorst = #match.opponents
-	}
-	Array.forEach(Array.reverse(match.opponents), function(opponent)
-		local place = opponent.placement
-		opponent.extradata.worstPlace = cache.currentWorst
-		if place == cache.index then
-			cache.currentWorst = place - 1
+			}
+		elseif column.id == 'opponent' then
+			column.header = {value = 'Participant'}
```

Comment:
Let's do it commons, bonus point to making it dynamic depending on opponent type

Reply:
not doing dynamic for now
- would need adjustments in how header is handled in general (make is a function based on match)
- the funtion would need to check if there are different opp types in the match.opponents and use `'Participants'` in that case anyways, only if all opp types are the same it would need to use the dynamic display